### DISCUSSION TITLE
Blade Sandbox Mode

### DISCUSSION THREAD

**Discussion by michael-rubel** [laravel/framework#44340] [Ideas]
Hello!

Are there any plans to support the sandbox mode in Blade templates as Twig does? I mean the possibility to allow users to edit Blade templates in a secure way. It'll probably be hard to maintain because Blade doesn't have many code restrictions as in Twig, but was such a feature considered?

See: https://twig.symfony.com/doc/3.x/tags/sandbox.html

---

**Comment by olsza** [comment]
Good idea

It would be nice if no variables could be read in `@sandbox`, unless you add in the `@sandbox` parameter what variables the user can use.
